rfc6962-bis.xml

```diff
@@ -464,6 +471,9 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
         <t>
           Note that the ASN.1 length and the opaque vector length are identical in size (1 byte) and value, so the DER encoding of the OID can be reproduced simply by prepending an OBJECT IDENTIFIER tag (0x06) to the opaque vector length and contents.
         </t>
+        <t>
+          OIDs used to identify logs are limited such that their DER encoding is less than 127 octets.
```

Comment:
Done, replaced with:
"such that the DER encoding of their value is less than 127 octets."